sdk/storage/azure-storage-common/src/main/java/module-info.java

```diff
@@ -7,4 +7,16 @@
     exports com.azure.storage.common;
     exports com.azure.storage.common.credentials;
     exports com.azure.storage.common.policy;
+
+    exports com.azure.storage.common.implementation.credentials to
```

Comment:
Add a `TODO` comment to remove these later.

Reply:
Done